src/NuGet.Clients/NuGet.PackageManagement.VisualStudio/Services/NuGetProjectUpgraderService.cs

```diff
@@ -40,7 +39,6 @@ public NuGetProjectUpgraderService(
             Assumes.NotNull(authorizationServiceClient);
             Assumes.NotNull(state);
 
-            _options = options;
```

Comment:
This parameter can also be removed from the constructor and its callers. This comment can be addressed in a follow-up PR.

Reply:
isnt that a braking change?

Reply:
We don't track public API for this project, so breaking changes are acceptable.